CHANGES.md

```diff
@@ -20,7 +20,8 @@
   * `miss_rate` in favor of `false_negative_rate`
   * `specificity_score` in favor of `true_negative_rate`
 * Change from public to private:
-  * `mean_{over,under}prediction` changed to `_mean_{over,under}prediction`
+  * `mean_{over,under}prediction` and `{balanced_,}root_mean_squared_error`
```

Comment:
were these public pre-0.4.6? Because if these were just introduced and already made private all within the work leading up to 0.4.6 then there's no need to mention that.

Reply:
Yes, they were public before, and they're used in the dashboard.